lib/ohai/plugins/habitat.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+#
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+Ohai.plugin(:Habitat) do
+  provides "habitat"
+
+  def fetch_habitat_version
+    shell_out("hab -V").stdout.gsub(/hab\s*/, "").strip
```

Comment:
Do you mean to parse the output as an array and select the field I want rather than gsub'ing the string data?  I've never seen an example of `shell_out()` with an array passed for a single command with an argument so just wanting to clarify.  The rest of the plugins in this repo pass the binary and arguments as a string as well -- or are you wanting to change moving forward?

Reply:
```suggestion
    shell_out(["hab", "-V"]).stdout.gsub(/hab\s*/, "").strip
```
The shellout helper wants arrays so we can avoid OS specific spacing terribleness.